### DISCUSSION TITLE
Numactl's memory bytes value is strange

### DISCUSSION THREAD

**Discussion by cwk08** [opcm/pcm#330] [Q&A] [answered]
I tried to use pcm-memory.cpp to test persistent memory's read bytes and write bytes on my code. but when I add numactl instruction at execution, the result I get is very wired, I found that  with numactl added, the bytes value is much lower than the original result without any numactl instruction. Two test uses completely same configuration except for an extra numactl instruction, is there any rational explain for this phenomenon ?

**Comment by opcm** [comment, marked as answer]
there can be additional overhead traffic to manage coherency between sockets if the data access is not NUMA local.
